dvc/remote/local/__init__.py

```diff
@@ -565,4 +556,6 @@ def unprotect(path_info):
 
     @staticmethod
     def protect(path_info):
```

Comment:
not related to this PR: it's not static in BASE

Reply:
This will also get lost here

Reply:
It is going to be fixed in #2090